### ISSUE TITLE
1.5.0 docker-compose up has --no-build by default

### ISSUE THREAD

**Issue by tarikihm** [docker/compose#2319]
After upgrading to docker toolbox 1.9.0/compose 1.5.0, `docker-compose up` will no longer build containers:

```
Tarik-MacBook-Pro:web tarikansari$ docker-compose up
Recreating 80f34a2a35_80f34a2a35_80f34a2a35_80f34a2a35_80f34a2a35_web_web_1
ERROR: Service 'web' needs to be built, but --no-build was passed.
Tarik-MacBook-Pro:web tarikansari$ docker-compose -v
docker-compose version: 1.5.0
```

**Comment by dnephin** [comment]
I think this is a duplicate of #1679, there are reports of this pre-1.5.0. See https://github.com/docker/compose/issues/1679#issuecomment-151291634 for a workaround.

Working on a fix for this in #2314

Let me know if the workaround doesn't work for you

**Comment by tarikihm** [comment]
Thanks for pointing this out.
